### PR TITLE
11.0 mig product supplierinfo discount

### PR DESCRIPTION
**Pull request by DarkoNikolovski**

### CODE REVIEW

File: product_supplierinfo_discount/models/__init__.py [hunk 1]
```diff
@@ -1,9 +1,8 @@
-# -*- coding: utf-8 -*-
 # © 2014 Serv. Tecnol. Avanzados (http://www.serviciosbaeza.com)
 #        Pedro M. Baeza <pedro.baeza@serviciosbaeza.com>
 # © 2016 ACSONE SA/NV (<http://acsone.eu>)
 # License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
 from . import product_supplierinfo
 from . import purchase_order
-from . import procurement_order
+from . import procurement
```
Comment (legalsylvain) [inline product_supplierinfo_discount/models/__init__.py]: why don't you rename to procurement_rule file ?

File: product_supplierinfo_discount/models/procurement.py [hunk 2]
```diff
@@ -1,27 +1,28 @@
-# -*- coding: utf-8 -*-
 # © 2016 ACSONE SA/NV (<http://acsone.eu>)
 # License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
 from odoo import api, fields, models
 
 
 class ProcurementOrder(models.Model):
```
Comment (legalsylvain) [inline product_supplierinfo_discount/models/procurement.py]: should be renamed into ProcurementRule.

File: product_supplierinfo_discount/views/product_supplierinfo_view.xml [hunk 3]
```diff
@@ -1,23 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
 <odoo>
+<record model="ir.ui.view" id="product_supplierinfo_form_view">
```
Comment (legalsylvain) [inline product_supplierinfo_discount/views/product_supplierinfo_view.xml]: please do not change tab. (that are correct BTW). It make the PR harder to review.

File: product_supplierinfo_discount/views/res_partner_view.xml [hunk 4]
```diff
@@ -1,18 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
 <odoo>
-
-    <record model="ir.ui.view" id="res_partner_form_view">
```
Comment (legalsylvain) [inline product_supplierinfo_discount/views/res_partner_view.xml]: same here.

**Review by legalsylvain — COMMENTED** [review]
some remarks inline.
thanks for the work.

File: product_supplierinfo_discount/models/__init__.py (hunk 1 above)
Comment (DarkoNikolovski) [reply product_supplierinfo_discount/models/__init__.py]: I renamed it to procurement because in the odoo's code base it is named procurement.

Comment (legalsylvain) [reply product_supplierinfo_discount/models/__init__.py]: yes. Well in OCA modules, rules are more precise. You can check them here : 
https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#file-naming

Comment (DarkoNikolovski) [reply product_supplierinfo_discount/models/__init__.py]: No problem, I will rename it :)

File: product_supplierinfo_discount/__init__.py [hunk 5]
```diff
@@ -0,0 +1,5 @@
+# © 2014 Serv. Tecnol. Avanzados (http://www.serviciosbaeza.com)
```
Comment (nikul-serpentcs) [inline product_supplierinfo_discount/__init__.py:1]: @DarkoNikolovski  IMO  remove Copyright in all __init__.py file

File: product_supplierinfo_discount/__manifest__.py [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+{
+    "name": "Discounts in product supplier info",
+    "version": "11.0.1.0.0",
+    "author": "Tecnativa, "
+              "GRAP, "
+              "Odoo Community Association (OCA)",
+    "category": "Purchase Management",
+    "website": "http://www.serviciosbaeza.com/",
```
Comment (nikul-serpentcs) [inline product_supplierinfo_discount/__manifest__.py:15]: @DarkoNikolovski Improve URL as per [this](https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L10)

File: product_supplierinfo_discount/views/product_supplierinfo_view.xml [hunk 7]
```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record model="ir.ui.view" id="product_supplierinfo_form_view">
+         <field name="model">product.supplierinfo</field>
+         <field name="inherit_id" ref="product.product_supplierinfo_form_view"/>
+         <field name="arch" type="xml">
+              <xpath expr="//div[field[@name='price']]" position="after">
```
Comment (nikul-serpentcs) [inline product_supplierinfo_discount/views/product_supplierinfo_view.xml:8]: @DarkoNikolovski Use directly `field`.

**Review by nikul-serpentcs — COMMENTED** [review]
Improve Code

**Comment by pedrobaeza** [comment]
Superseeded by #592 as this is all red.

**Pull request closed without merging**